utils/strings.js

```diff
@@ -1,7 +1,18 @@
 const _ = require('lodash');
 
 module.exports = {
-  pascalCase(input) {
-    return _.chain(input).camelCase().upperFirst().value();
+  pascalCase(input, upperFirst = true) {
```

Comment:
This is "PascalCase" and this is "camelCase".
So we should not expose a method that is named `pascalCase` but has an argument to create generate camel cased strings.

```javascript
pascalCase(input) {
  return _.chain(input).camelCase().upperFirst().value();
}

camelCase(input) {
   return _.camelCase(value);
}
```
is much better in my opinion.

Reply:
oh yes my bad.